README.rst

```diff
@@ -41,21 +89,58 @@ and running the following in the astropy-benchmarks folder::
 
 This will run all the benchmarks in fast mode (running each function once) using the installed version of Astropy. This is just to make sure that the benchmarks run properly, and the timings should not be considered accurate.
 
-To run a single benchmark, you can do e.g.::
+You can select just the benchmark you have written using the ``--bench`` option::
 
     asv dev --bench time_my_benchmark
 
-Once you are happy with your benchmarks, open a pull request to the astropy-benchmarks repository.
+Running benchmarks against a local commit
+-----------------------------------------
+
+If you are trying to improve the performance of astropy and you have made some
+commits in your local repository that you want to test before opening a pull
+request to astropy (or before a pull request is merged). To do this, we need
+to switch from using the upstream repository to the repository on your computer,
+which is done by editing the ``asv.conf.json`` file and finding the following
+section::
+
+    // The git URL to the project being tested. Comment the first line
+    // and uncomment and edit the second if you are testing local changes.
+    "repo": "https://github.com/astropy/astropy.git",
+    //"repo": "/your/local/repository/”,
+
+Comment out the first “repo” line and uncomment the second, replacing the path
+with the absolute path to your local clone of Astropy. You will then be able to
+run the benchmarks for a commit in your local repository using e.g.::
+
+    asv run 827f322b^!
+
+Comparing commits
+-----------------
+
+If you want to compare two commits (e.g. the latest upstream commit and a local
+commit), you can use e.g.::
+
+    asv compare 88fbbc33 827f322b
+
+This will show a table with a comparison of the benchmark times for the two
+commits.
+
+Contributing benchmarks
+-----------------------
 
-You do not need to add any result files for the benchmarks - we have machines that automatically do this every night.
+Once you are happy with your benchmark(s), open a pull request to the
+astropy-benchmarks repository. You do not need to add any result files for the
+benchmarks - we have machines that automatically do this every night.
```

Comment:
Please add a short note about how to view the results, even if it's just pointing to the relevant asv docs pages ( https://asv.readthedocs.io/en/stable/using.html#viewing-the-results)

Reply:
Done